CHANGELOG.md

```diff
@@ -81,11 +93,11 @@ No changes, only changed to a released version of the `thrift` crate dependency.
 
 * Adjusted precedence of revset union/intersection/difference operators.
   `x | y & z` is now equivalent to `x | (y & z)`.
-  
+
```

Comment:
Nit: split whitespace cleanup to new commit (if that doesn't bother you much.)

Reply:
I'm gonna skip this one (mostly out of laziness of reverting, will separate from next time).